fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -237,34 +237,34 @@
 			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
 			SPECIES_PERK_ICON = "handshake",
 			SPECIES_PERK_NAME = "Beefy Limbs",
-			SPECIES_PERK_DESC = "Beefmen are able to tear off and put limbs back on at will. They do this by targetting their limb and right clicking.",
+			SPECIES_PERK_DESC = "Beefmen are able to tear off limbs and put them back at will. This can be done by targetting a limb and right clicking with an empty hand.",
 		),
 		list(
 			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
 			SPECIES_PERK_ICON = "running",
 			SPECIES_PERK_NAME = "Runners",
-			SPECIES_PERK_DESC = "Beefmen are 20% faster than most other species.",
+			SPECIES_PERK_DESC = "Beefmen are 20% faster than most humanoids.",
 		),
 		list(
 			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
 			SPECIES_PERK_ICON = "temperature-low",
 			SPECIES_PERK_NAME = "Cold Loving",
-			SPECIES_PERK_DESC = "Beefmen are completely immune to the cold. Low temperatures even prevent bleeding.",
+			SPECIES_PERK_DESC = "Beefmen are completely immune to the cold. Low temperatures even prevent them from bleeding.",
 		),
 		//Neutral
 		list(
 			SPECIES_PERK_TYPE = SPECIES_NEUTRAL_PERK,
 			SPECIES_PERK_ICON = "link",
 			SPECIES_PERK_NAME = "Phobetor Tears",
-			SPECIES_PERK_DESC = "Beefmen can see and use Phobetor tears, small tears in reality, \
-				that teleport them to the other end of the tear. This can only be done when no one is watching.",
+			SPECIES_PERK_DESC = "Beefmen can see and use Phobetor Tears, small tears in reality \
+				that allow them to teleport between random locations. This can only be done when they are unobserved.",
```

Comment:
It's not a random location beefmen can see where the end of the tear is and can go back and forth

Reply:
I'll clarify that.